test/ctia/bulk/core_test.clj

```diff
@@ -334,14 +336,14 @@
   (let [intermediate-tempids (atom [])
         incident1 (assoc incident-minimal :id "transientid1")
         incident2 (assoc incident-minimal :id "transientid2")]
-    (is (= {:bulk-refs {:incidents [incident1 incident2]}
+    (is (= {:bulk-refs {:incidents (mapv :id [incident1 incident2])}
```

Comment:
this a change of behaviour, did we have a bug?

Reply:
Mysterious, will circle back to this.

Reply:
I changed the test a few lines down vvvv to satisfy a schema that was previously unenforced. This line ^^^^ also needed to change because the result changed.